packages/api/src/hooks/index.ts

```diff
@@ -35,6 +35,7 @@ export { default as useDxtradeServiceToken } from './useDxtradeServiceToken';
 export { default as useDynamicLeverage } from './useDynamicLeverage';
 export { default as useExchangeRate } from './useExchangeRate';
 export { default as useGetAccountStatus } from './useGetAccountStatus';
+export { default as useIdentityDocumentVerificationAdd } from './useIdentityDocumentVerificationAdd';
```

Comment:
We need to use this hook to submit the IDV:
<img width="1840" alt="image" src="https://github.com/binary-com/deriv-app/assets/103016120/fb5842b2-548c-4712-8543-17f32ba88408">

Reply:
<img width="1840" alt="Screenshot 2023-11-24 at 1 05 11 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/3633f66f-2173-42f7-9e0b-26d0b36da750">